code/game/objects/items/storage/backpack.dm

```diff
@@ -27,6 +29,46 @@
 	if (..() && use_sound)
 		playsound(loc, use_sound, 15, TRUE, 6)
 
+/obj/item/storage/backpack/attack(mob/living/target_mob, mob/living/user)
+	if(!xeno_icon_state)
+		return ..()
+	if(!isXeno(target_mob))
+		return ..()
+	if(target_mob.back)
+		return ..()
+	if(user.a_intent != INTENT_HELP)
+		return ..()
+	if(!xeno_types || !(target_mob.type in xeno_types))
+		return ..()
+	
```

Comment:
architect requirement : you must only be able to strap backpacks onto xenos that your faction is allied with. Attempting to do so with non allied xenos must make them stun you like when trying to grab them.

Reply:
boo